pkg/controllers/externalsecret/externalsecret_controller.go

```diff
@@ -52,6 +52,7 @@ type Reconciler struct {
 	ControllerClass string
 }
 
+// Reconcile returns a new ExternalSecret object.
```

Comment:
I would change this to something like:
```
// Reconcile implements the main reconciliation loop 
// for watched objects (ExternalSecret, ClusterSecretStore and SecretStore), 
// and updates/creates a Kubernetes secret based on them
```

Reply:
I am not sure how the linter handles multiline comments for docstrings like this, try to check it out with `make fmt`